server/run-tests-with-grcov.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+# cleanly run the following commands in their own session
+bash -c "
+# tell Rust to run with coverage instrumentation
+RUSTFLAGS=\"-Cinstrument-coverage\"
+# give grcov a profile name template for output files
+LLVM_PROFILE_FILE=\"svix-webhooks-%p-%m.profraw\"
+# put the compiler in nightly mode
+RUSTC_BOOTSTRAP=1
+
+# run tests
+./run-tests.sh
+" || true
```

Comment:
Why are you doing this in a subscript like this?

Reply:
So that these variables are only set in this new session, and don't affect rustc in any subsequent commands a developer might run unrelated to this script.

Reply:
You don't need to do anything, it won't affect anything outside of the script.
Also, I just noticed you didn't run `export` so actually these won't even work at all.